song-importer/src/main/java/org/icgc/dcc/song/importer/convert/AnalysisConverter.java

```diff
@@ -126,15 +125,19 @@ private static SequencingReadAnalysis buildSequencingReadAnalysis(SeqReadAggrega
   }
 
   private static void updateAnalysis(Analysis analysis, SeqReadAggregate seqReadAggregate){
-    analysis.setAnalysisId(seqReadAggregate.getAnalysisId());
+    val analysisId = seqReadAggregate.getAnalysisId();
```

Comment:
nothing to change here at the moment, but using methods for updating objects through side effect is pretty bad. The void return means you can't chain calls or use properly in a stream.